### PR TITLE
packagegroup-dts: Add remaining recipes required to use tpm2_tools

### PR DESCRIPTION
**Pull request by PLangowski**

### CODE REVIEW

File: meta-dts-distro/recipes-dts/packagegroups/packagegroup-dts.bb [hunk 1]
```diff
@@ -41,6 +44,15 @@ RDEPENDS:${PN}-tools-base = " \
...
     tpm2-tools \
+    packagegroup-tpm2 \
+    packagegroup-tpm2-initramfs \
+    tpm2-abrmd \
+    tpm2-tss \
+    libtss2 \
+    libtss2-mu \
+    libtss2-tcti-device \
+    libtss2-tcti-mssim \
+    tpm2-abrmd \
```
Comment (miczyg1) [inline meta-dts-distro/recipes-dts/packagegroups/packagegroup-dts.bb:55]: tpm2-abrmd is listed twice

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: meta-dts-distro/recipes-dts/packagegroups/packagegroup-dts.bb [hunk 2]
```diff
@@ -41,6 +44,15 @@ RDEPENDS:${PN}-tools-base = " \
 
 RDEPENDS:${PN}-tools-dts = " \
     tpm2-tools \
+    packagegroup-tpm2 \
```
Comment (TomaszAIR) [inline meta-dts-distro/recipes-dts/packagegroups/packagegroup-dts.bb:47]: @PLangowski are those two packagegroup (tpm2 and tpm2-initramfs) needed? What they provide?

File: meta-dts-distro/recipes-dts/packagegroups/packagegroup-dts.bb (hunk 1 above)
Comment (PLangowski) [reply meta-dts-distro/recipes-dts/packagegroups/packagegroup-dts.bb]: fixed 69264d7c54955ac7aaaed830898fb1f3b0ec193d

File: meta-dts-distro/recipes-dts/packagegroups/packagegroup-dts.bb (hunk 2 above)
Comment (PLangowski) [reply meta-dts-distro/recipes-dts/packagegroups/packagegroup-dts.bb:46]: After looking into the source layer I remove unnecessary duplicates 69264d7c54955ac7aaaed830898fb1f3b0ec193d

**Comment by TomaszAIR** [comment]
@PLangowski I switched used packagegroup to the one from meta-security layer which is a layer that we use for dts standard image (without UEFI Secure Boot integrated). Merging.

Proof:

```
root@DasharoToolsSuite:~# tpm2_getcap properties-variable
TPM2_PT_PERMANENT:
  ownerAuthSet:              0
  endorsementAuthSet:        0
  lockoutAuthSet:            1
  reserved1:                 0
  disableClear:              0
  inLockout:                 0
  tpmGeneratedEPS:           0
  reserved2:                 0
TPM2_PT_STARTUP_CLEAR:
  phEnable:                  1
  shEnable:                  1
  ehEnable:                  1
  phEnableNV:                1
  reserved1:                 0
  orderly:                   0
TPM2_PT_HR_NV_INDEX: 0x7
TPM2_PT_HR_LOADED: 0x0
TPM2_PT_HR_LOADED_AVAIL: 0x3
TPM2_PT_HR_ACTIVE: 0x0
TPM2_PT_HR_ACTIVE_AVAIL: 0x40
TPM2_PT_HR_TRANSIENT_AVAIL: 0x3
TPM2_PT_HR_PERSISTENT: 0x6
TPM2_PT_HR_PERSISTENT_AVAIL: 0xB
TPM2_PT_NV_COUNTERS: 0x2
TPM2_PT_NV_COUNTERS_AVAIL: 0xC
TPM2_PT_ALGORITHM_SET: 0x0
TPM2_PT_LOADED_CURVES: 0x2
TPM2_PT_LOCKOUT_COUNTER: 0x0
TPM2_PT_MAX_AUTH_FAIL: 0x1F
TPM2_PT_LOCKOUT_INTERVAL: 0x258
TPM2_PT_LOCKOUT_RECOVERY: 0x15180
TPM2_PT_AUDIT_COUNTER_0: 0x0
TPM2_PT_AUDIT_COUNTER_1: 0x0
```

**Pull request merged**